xmlmemory.c

```diff
@@ -978,13 +978,13 @@ xmlInitMemory(void)
      xmlMemInitialized = 1;
      xmlMemMutex = xmlNewMutex();
 
-#ifdef HAVE_STDLIB_H
+#ifdef defined(HAVE_STDLIB_H) && !defined(__ORBIS__)
```

Comment:
Could just include YiPort.h there is a getenv in there.

Reply:
`YiPort.h` is not available, as this is build as a standalone library

Reply:
fixed